### PR TITLE
Resolve vulnerabilities from dependencies

### PR DESCRIPTION
**Pull request by MuneebAijaz**
- Resolving vulnerabilities from dependencies in both backend and frontend
- Changing workflows to test and push PR images on certain labels

### CODE REVIEW

File: .github/workflows/run-tests-on-pr copy.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: Run Tests on PR Label
```
Comment (karl-johan-grahn) [inline .github/workflows/run-tests-on-pr copy.yaml:1]: Remove this copy file

**Review by karl-johan-grahn — CHANGES_REQUESTED** [review]

File: .github/workflows/run-tests-on-pr copy.yaml (hunk 1 above)
Comment (karl-johan-grahn) [reply .github/workflows/run-tests-on-pr copy.yaml:1]: Keep it but remove ` copy` from filename

File: .github/workflows/pull_request.yaml [hunk 2]
```diff
@@ -38,36 +38,14 @@ jobs:
...
-        go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.56.0
+        go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.61.0
         golangci-lint run --timeout=10m ./...
 
     - name: Helm Lint
       run: |
         cd deployments/kubernetes/chart/forecastle
         helm lint
 
-    # - name: Install kubectl
```
Comment (karl-johan-grahn) [inline .github/workflows/pull_request.yaml:49]: Why commenting out this? Why not just remove it since it was move to run-tests-on-pr

Comment (MuneebAijaz) [reply .github/workflows/pull_request.yaml:49]: you are looking at a removed change :smile:

File: .github/workflows/run-tests-on-pr copy.yaml (hunk 1 above)
Comment (MuneebAijaz) [reply .github/workflows/run-tests-on-pr copy.yaml]: done

**Review by karl-johan-grahn — APPROVED** [review]

**Pull request merged**